packit_service/service/events.py

```diff
@@ -58,9 +57,31 @@ class WhitelistStatus(enum.Enum):
 class Event:
     def __init__(self, trigger: JobTriggerType):
         self.trigger: JobTriggerType = trigger
+        self._user_config = None
+
+    @property
+    def user_config(self):
```

Comment:
user_config is going to be replaced by service_config in #60

Reply:
Do I need to fix that here?

Reply:
no need, I will do while rebasing my PR.